assets/js/modules/analytics/components/common/UseSnippetSwitch.js

```diff
@@ -52,7 +63,13 @@ export default function UseSnippetSwitch() {
 		);
 	}, [ useSnippet, setUseSnippet, viewContext ] );
 
-	if ( useSnippet === undefined ) {
+	useEffect( () => {
+		if ( toggleSwitchDisabled ) {
+			setUseSnippet( false );
+		}
+	}, [ setUseSnippet, toggleSwitchDisabled ] );
```

Comment:
Can you explain this change? Why do we set the snippet to `false` here? That seems new and isn't specifically mentioned in the ACs (or IB) as far as I can tell.

Reply:
Yeah, this wasn't mentioned in the ACs, but if the switch to use our snippet is disabled, I thought it best to disable our snippet. Otherwise, if the snippet were disabled but the setting was previously set to `true`, then the user would be forced to output our snippet _and_ the existing snippet, with no way to disable ours because the snippet setting cannot be changed.

But actually I think this needs some further thought, you're right 👍🏻